.env-sample

```diff
@@ -16,9 +16,7 @@ CACHE_REDIS_CONNECTION_STRING=
 # the URLs should refer to your UI implementation's reset password, email validation, and container invite pages
 EMAIL_ADDRESS=do+not+reply@deeplynx.org
 EMAIL_ENABLED=true
-EMAIL_VALIDATION_URL=http://localhost:8080/validate
```

Comment:
What happened to these email values?

Reply:
These used to be needed in order to format the email invitation to direct the users to the UI. Since Deep Lynx now has it's own UI it's no longer needed.